wvdecrypter/wvdecrypter.cpp

```diff
@@ -469,7 +471,7 @@ WV_CencSingleSampleDecrypter::WV_CencSingleSampleDecrypter(WV_DRM &drm, AP4_Data
     reinterpret_cast<const uint8_t *>(pssh_.GetData()), pssh_.GetDataSize());
 
   int retrycount=0;
-  while (session_.empty() && ++retrycount < 100)
+  while (!drm.GetCdmAdapter()->IsSessionActive() && ++retrycount < 100)
```

Comment:
Also not for this PR but this is not a good way to this. A wait for condition would be must better. Then no need to thread::sleep_for